lint/style.py

```diff
@@ -46,16 +43,6 @@ def load_gutter_icons():
         colorize = True
 
     new_gutter_dict["colorize"] = colorize
-    dir_path, _ = os.path.split(theme_file)
-    pck_path = sublime.packages_path().split("/Packages")[0]
```

Comment:
To make the bug clearer. This is broken, the original author meant probably `os.path.dirname`.